recipes/mamba-ssm/recipe.yaml

```diff
@@ -0,0 +1,65 @@
+context:
+  name: mamba-ssm
+  version: 2.2.5
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/state-spaces/mamba/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 1e072841619af9c171709b2c6ff17daa0d9bf7caf0fcb083f5c11754b63c6cdb
+
+build:
+  number: 0
+  script:
+    env:
+      MAMBA_FORCE_BUILD: "TRUE"
+    content: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip:
+    # mamba-ssm requires CUDA (or ROCm) GPU
+    - cuda_compiler_version == "None"
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ compiler('cuda') }}
+    - ${{ stdlib('c') }}
+  host:
+    - cuda-version ==${{ cuda_compiler_version }}
+    - ninja
+    - packaging
+    - pip
+    - python
+    - pytorch * cuda*
+    - setuptools >=61.0.0
+    # Remove triton<3.2.0 pin when pytorch depends on triton>=3.4.0
+    # containing the https://github.com/triton-lang/triton/pull/6570 bugfix
+    - triton <3.2.0
+    - wheel
+  run:
+    - einops
+    - packaging
+    - python
+    - pytorch
+    - setuptools >=61.0.0
+    - transformers
+
+tests:
+  - python:
+      imports:
+        - mamba_ssm
+      # disabled pip check because ninja is not available
+      # see https://github.com/conda-forge/ninja-feedstock/issues/26
```

Comment:
Can we add `ninja` as a test-only dependency here?

Reply:
Don't think there's a way to add test-only dependencies with v1 recipes - https://rattler.build/latest/reference/recipe_file/#python-tests, only able to set the python-version.

Reply:
Ok, I've added the patch, and Python 3.10 build's `pip check` seems to have passed. Just need to wait for the other matrix CI builds to finish.